src/platforms/rp2040/src/main.c

```diff
@@ -24,6 +24,7 @@
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wpedantic"
 
+#include "pico/bootrom.h"
```

Comment:
Could this include be styled and sorted like the others in this file?

Reply:
Oops! Fixed.